comfy/latent_formats.py

```diff
@@ -37,6 +37,20 @@ def __init__(self):
                 ]
         self.taesd_decoder_name = "taesdxl_decoder"
 
+class HunyuanDit(LatentFormat):
```

Comment:
Why is the separate latent format needed? This is just a copypaste of sdxl

Reply:
I had removed it.